axelar-chains-config/info/devnet-amplifier.json

```
@@ -114,6 +144,21 @@
       "finality": "finalized",
       "approxFinalityWaitTime": 40,
       "contracts": {
+        "AxelarGateway": {
+          "deployer": "0xC5C73C3032e0577662000887D60d279B5Cc1C1AB",
+          "startingKeyIDs": [
+            "core-optimism-genesis"
+          ],
+          "address": "0xdb6711cFc97e13E4aF6EEEe5f4923A9c2FBf0721",
+          "implementation": "0x6BB7937D233e3Ea48C231114cEc91739aAfa2021",
+          "implementationCodehash": "0xcc7160c968bc78c8be295b69fce8725ca9813669fc96bf460c5c5c1e13b2bf3d",
```

```suggestion
          "implementationCodehash": "0xfa1b1e1e8e971ce57ab86a74fbd8a92b15b3bd8832c300db0fae6e688a4c6a59",
```
this is what i computed via:
```
curl --request POST \
     --url https://sepolia.optimism.io \
     --header 'accept: application/json' \
     --header 'content-type: application/json' \
     --data '
{
     "id": 1,
     "jsonrpc": "2.0",
     "params": [
          "0x6BB7937D233e3Ea48C231114cEc91739aAfa2021",
          "latest"
     ],
     "method": "eth_getCode"
}' | jq .result
```
and then hashing it
https://emn178.github.io/online-tools/keccak_256.html